### ISSUE TITLE
Not for AVR

### ISSUE THREAD

**Issue by CombiesGit** [durydevelop/arduino-lib-oled#9]
> architectures=*

```
oled.h:306:18: error: no members matching 'Print::printf' in 'class Print'

     using Print::printf;

                  ^~~~~~
```

**Comment by sixtyfive** [comment]
Yup, I was just burned by that, too ... shame :-(

**Comment by MrMonkeyKing** [comment]
No one found a fix for that yet?

**Comment by durydevelop** [comment]
Fixed in 1.0.6
